docs/documentation.rst

```diff
@@ -0,0 +1,70 @@
+Documentation and Tutorials
+===========================
+
+Parcels has several documentation and tutorial Jupyter notebooks which go through various aspects of parcels. Static versions of these notebooks are available below in the site, with the interactive notebooks being available here :download:`parcels_tutorials.zip </_downloads/parcels_tutorials.zip>`.
+
+
+
+.. nbgallery::
+   :caption: Overview
+   :name: tutorial-overview
+
+   examples/tutorial_parcels_structure.ipynb
+   examples/parcels_tutorial.ipynb
+   examples/tutorial_output.ipynb
+
+
+.. nbgallery::
+   :caption: Setting up FieldSets
+   :name: tutorial-fieldsets
+
+   examples/tutorial_nemo_curvilinear.ipynb
+   examples/tutorial_nemo_3D.ipynb
+   examples/tutorial_SummedFields.ipynb
+   examples/tutorial_NestedFields.ipynb
+   examples/tutorial_timevaryingdepthdimensions.ipynb
+   examples/tutorial_periodic_boundaries.ipynb
+   examples/tutorial_interpolation.ipynb
+   examples/tutorial_unitconverters.ipynb
+   examples/tutorial_timestamps.ipynb
+   examples/documentation_indexing.ipynb
+
+
+.. nbgallery::
+   :caption: Creating ParticleSets
+   :name: tutorial-particlesets
+
+   examples/tutorial_jit_vs_scipy.ipynb
+   examples/tutorial_delaystart.ipynb
+
+
+.. nbgallery::
+   :caption: Writing kernels to be executed on each particle
+   :name: tutorial-kernels
+
+   examples/tutorial_diffusion.ipynb
+   examples/tutorial_sampling.ipynb
+   examples/tutorial_particle_field_interaction.ipynb
+   examples/tutorial_interaction.ipynb
+   examples/tutorial_analyticaladvection.ipynb
+
+
+.. nbgallery::
+   :caption: Other tutorials
+   :name: tutorial-other
+
+   examples/documentation_MPI.ipynb
+   examples/documentation_stuck_particles.ipynb
+   examples/documentation_unstuck_Agrid.ipynb
+   examples/documentation_LargeRunsOutput.ipynb
```

Comment:
This one also still needs a thumbnail. I don't really have an idea what could work here. Any suggestions?

Reply:
https://upload.wikimedia.org/wikipedia/commons/7/70/Oxygen480-devices-drive-harddisk.svg ?
![](https://upload.wikimedia.org/wikipedia/commons/7/70/Oxygen480-devices-drive-harddisk.svg)